lakefile.lean

```diff
@@ -10,13 +10,21 @@ package cvc5 {
   precompileModules := true
   moreGlobalServerArgs := #[s!"--load-dynlib={libcpp}"]
   extraDepTargets := #[`libcvc5]
+  testDriver := "cvc5Test"
 }
 
 @[default_target]
 lean_lib cvc5 {
   moreLeanArgs := #[s!"--load-dynlib={libcpp}"]
 }
 
+lean_lib cvc5Test {
+  globs := #[Glob.submodules `Cvc5Test]
+}
```

Comment:
Could you try creating a new file in the root directory called `Test.lean`, rename the directory `Cvc5Test` to `Test`, and import all tests from there? Then, maybe this would be enough
```suggestion
@[test_driver]
lean_lib Test
```

Reply:
`Test` would not be special library, I'm pretty sure if we did that we would have a conflict with other packages with a `Test` library.

About `globs`, I did that to *i)* pollute the top-level as little as possible and *ii)* not having to list all tests in `Cvc5Test.lean`, to prevent from forgetting to add new tests to the list.

Reply:
Do you still want to make both/either of these changes?

Reply:
No need. We can change the name once https://github.com/leanprover/lean4/issues/4168 is resolved.